.circleci/config.yml

```diff
@@ -12,11 +18,10 @@ jobs:
       - checkout
       - restore_cache:
           keys:
-            - v3-dependencies-{{ checksum "Pipfile.lock" }}
-      - run: &install-pipenv pip install -U pipenv pip --quiet --no-input
-      - run: pipenv sync --dev
-      - run: pipenv run pipenv check # before save_cache so an insecure cache is never saved
-      - run: pipenv requirements > postcode_lookup/requirements.txt
+            - v4-dependencies-{{ checksum "uv.lock" }}
+      - install_uv
+      - run: uv sync --dev --no-python-downloads
```

Comment:
Could we run
`uv lock --locked`
(this will throw an error if the lockfile is out of sync)
here.

Reply:
mashed this into https://github.com/DemocracyClub/ec-postcode-lookup-pages/pull/117/commits/7cab83a01f204b6852297e55d2bc3901ee232f14
whoops